bool mirror
bool dynamical
bool ballSpecial
uint8 bMotionType   # ENUM(BMotionID: kickForward, newKick, none)


DynPoint[2] dynPoints

